eds/blocks/local-navigation/local-navigation.css

```diff
@@ -50,15 +50,20 @@
   align-items: center;
   text-decoration: none;
   background-color: #202020;
-  padding-inline: var(--space-8);
+  padding-inline: var(--space-6);
 }
 
-.local-navigation ul li a[aria-current='true']:hover, .local-navigation ul li.page-title a:hover {
+.local-navigation ul li a[aria-current='true']:hover, .local-navigation ul li a:hover {
```

Comment:
Shorted up with `:is()` since everything is the same, just adding two selectors (true and a)